redbox/models/settings.py

```diff
@@ -81,6 +82,17 @@ def s3_client(self):
             )
 
         elif self.object_store == "s3":
+            client = boto3.client(
+                "s3",
+                aws_access_key_id=self.aws_access_key_id,
+                aws_secret_access_key=self.aws_secret_access_key,
+                region_name=self.aws_region,
+            )
+        elif self.object_store == "moto":
+            from moto import mock_aws
```

Comment:
as this is a `dev` requirement we want to load this dynamically rather than adding it to the main poetry requirements